src/main/java/calculator/domain/Division.java

```diff
@@ -0,0 +1,19 @@
+package calculator.domain;
+
+public class Division implements Operation {
+    @Override
+    public int calculate(int a, int b) {
+        validateDivision(a, b);
+        return a / b;
+    }
+
+    private static void validateDivision(int a, int b) {
+        if (b == 0) {
+            throw new IllegalArgumentException();
```

Comment:
어떤 예외인지 메시지를 전달하는 것이 좋습니다.
그래야 빠른 파악이 가능합니다.
```suggestion
            throw new IllegalArgumentException("메시지");
```